tools/clang/unittests/HLSLTestLib/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+
+if(WIN32)
+find_package(TAEF REQUIRED)
+include_directories(${TAEF_INCLUDE_DIRS})
+
+add_clang_library(HLSLTestLib
+  D3DReflectionDumper.cpp
+  DxcTestUtils.cpp
+  FileCheckerTest.cpp
+  FileCheckForTest.cpp
+)
+else(WIN32)
+set(HLSL_IGNORE_SOURCES
+  D3DReflectionDumper.cpp
+  FileCheckerTest.cpp
+)
+add_clang_library(HLSLTestLib
+  DxcTestUtils.cpp
+  FileCheckForTest.cpp
+)
```

Comment:
The Linux version of DxcTestUtils.cpp uses headers from googletest (which works remarkably similarly to TAEF. what are the chances?). You can enable this by adding these lines right here:

include_directories(${DXC_GTEST_DIR}/googletest/include)
include_directories(${DXC_GTEST_DIR}/googlemock/include)